pkg/jsx/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@hyperapp/jsx",
+  "description": "Hyperapp JSX runtime",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "jsx-runtime.js",
```

Comment:
Let's rename the file to `index.js` and use `index.js` as `main`.

Reply:
I named it that way because for a `importSource` of `my-lib-name`, the transform will be `my-lib-name/jsx-runtime`. 
Idea was that we would set `@hyperapp/jsx` as importSource, and `@hyperapp/jsx/jsx-runtime` would be imported.
Not sure this is the best way to do it tho 🤔 .

Reply:
Third-party project needs can't really impose how we name files in Hyperapp, so this should be `index.js` in principle.

Reply:
We can rename to index.js here but it would have to be moved to a file named `jsx-runtime` in a publishing step or something? If you look at https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform, 
![image](https://user-images.githubusercontent.com/14101189/105637579-4a263a00-5e3c-11eb-99c3-5f4ba0cd4b74.png)

Reply:
It's published via @zaceno's hyperapp-jsx-pragma now and seems to work fine so far

Reply:
Thanks, I see now.